libraries/abstractions/secure_sockets/test/iot_test_tcp.c

```diff
@@ -2961,6 +2962,61 @@ TEST( Full_TCP, AFQP_SOCKETS_htons_HappyCase )
 }
 /*-----------------------------------------------------------*/
 
+TEST( Full_TCP, test_dns_cache_multiple_addresses )
```

Comment:
For consistency, can we name the test  `AFQP_SECURE_SOCKETS_TestDnsCacheMultipleAddresses`?

Reply:
The `AFQP` standards for [Amazon FreeRTOS Qualification Program](https://docs.aws.amazon.com/freertos/latest/qualificationguide/afr-qualification.html), and this test won't be added to that test suite now since that would be a qualification change.  I think it would make sense to add it to that test suite eventually, however this would have to be coordinated with that program.  We would change the name to an `AFQP` name at that time.

Reply:
That makes sense - however can we use camel casing instead of snake case?